frontend/src/views/reports/ReportingView.vue

```diff
@@ -265,6 +335,28 @@ export default {
       this.menuEndDate = false
       this.selectedFacility = this.currentFacility.facilityId
     },
+
+    async createSurveyResponse() {
+      try {
+        this.processing = true
+        const payload = {
+          contactId: this.userInfo?.contactId,
+          facilityId: this.selectedFacility,
+          surveyId: '16fb81de-6dc1-ee11-9079-000d3af4865d',
+          fiscalYearId: this.selectedFiscalYear?.fiscalYearId,
+          reportingMonthId: this.selectedReportingMonth?.monthId,
+          surveyResponseType: this.surveyResponseType,
+        }
+        const response = await ReportsService.createSurveyResponse(payload)
+        if (response?.surveyResponseId) {
+          this.$router.push({ name: 'survey-form', params: { surveyResponseGuid: response?.surveyResponseId } })
+        }
+      } catch (error) {
+        this.setFailureAlert('Failed to create new survey response', error)
```

Comment:
Maybe create New Monthly Report? I don't think we will be using the term Survey in the ui.